bindings/rust/wasmedge-sdk/examples/work_in_parallel.rs

```diff
@@ -0,0 +1,101 @@
+//! WasmEdge Runtime supports WebAssembly Threads proposal. This example demonstrates how to use multiple threads to
+//! render the [Mandelbrot set](https://en.wikipedia.org/wiki/Mandelbrot_set) in parallel.
+//!
+//! Use the following command to run the example:
+//!
+//! ```bash
+//! // cd bindings/rust
+//! cargo run -p wasmedge-sdk --example work_in_parallel -- --nocapture
+//! ```
+//!
+//! The file "mandelbrot.so" used in the code below is generated by `wasmedgec` command: `wasmedgec --enable-threads mandelbrot.wasm mandelbrot.so`. For details, please refer to [Enable AOT Mode](https://github.com/WasmEdge/WasmEdge/blob/master/examples/capi/mandelbrot-set-in-threads/README.md#enable-aot-mode).
```

Comment:
This line is tooooo long :-)

Reply:
The reason for the long description is that this example involves taking as input a file that needs users to generate by themselves, and also the result visualization is not part of the example. Hence, we need more space to explain these two parts so that they can understand where the input file comes from and how to check the computation result. That's why it looks a little bit "lengthy" :) Thanks!

Reply:
Got it. So we don't need to care about the line length inside the comments. Is it correct? I am not sure about the format of the rustdoc

Reply:
Yes, we don't need to care about rustdoc, as the rustdoc does not contain these examples.